docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -184,8 +184,7 @@ folder) you're ready to build and deploy the sample app.
 Once you have deployed the application and verified that the namespace, sample application and trigger are ready, let's send a CloudEvent.
```

Comment:
```suggestion
After you have deployed the application, and have verified that the namespace, sample application and trigger are ready, you can send a CloudEvent.
```
Stick with the active voice and no fluff :) https://developers.google.com/style/voice